ckanext/harvest/logic/action/harvest_additions.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# Purpose: Read CSV File and create harvest sources
+# Author: James Brown
+# ------------------------------------------
+
+# Display Help
+Help()
```

Comment:
Should be lowercase to be consistent with other functions.

Reply:
Fair.